src/Lib/Diagnostics/BehaviorPack/Item/components/diagnose.ts

```diff
@@ -52,7 +52,7 @@ const component_test: Record<string, ComponentCheck> = {
     if (Array.isArray(component.use_on)) component.use_on.forEach((block: string) => {
       behaviorpack_check_blockid(block, diagnoser)
     });
-    if (component.entity) behaviorpack_entityid_diagnose(component.entity, diagnoser)
+    if (component.entity) behaviorpack_entityid_diagnose(component.entity.split('<')[0], diagnoser)
```

Comment:
hmm, maybe it better if the behaviorpack_entityid_diagnose could deal with this 🤔 

Might not be the only time its valid, or will be valid

Reply:
I had considered this but figured since it was checking for the entity id only that should be passed. Will change